site/en/blog/style-queries/index.md

```diff
@@ -0,0 +1,231 @@
+---
+title: "Getting Started with Style Queries"
+description: >
+  Style queries allow developers to query a parent element's style values using the @container rule. In M112, style queries for CSS custom properties are landing stable. Learn how to get started with them.
+layout: 'layouts/blog-post.njk'
+date: 2023-02-23
+hero: 'image/HodOHWjMnbNw56hvNASHWSgZyAf2/N8vx7PDIeyMNI2wzWnax.jpg'
+alt: >
+  Two-tone pink and blue background with opposite colored purses on each background.
+tags:
+  - chrome-112
```

Comment:
this is in 111, have confirmed by testing the demo in current Beta. Chrome Status also lists 111.

Reply:
Yes, it's in 111! Good catch